test/runner.py

```diff
@@ -65,6 +65,8 @@
   'asan',
   'lsan',
   'ubsan',
+  'wasm64',
+  'wasm64_v8',
```

Comment:
Should this part also be reverted if the tests don't all pass?

Reply:
No, all tests so pass, just some are disabled.  We run the full test suites in CI